lib/CMakeLists.txt

```diff
@@ -57,6 +45,7 @@ add_library(prometheus-cpp
 
 # TODO(gj) make all PRIVATE
 target_link_libraries(prometheus-cpp PUBLIC ${PROTOBUF_LIBRARIES})
+target_include_directories(prometheus-cpp PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include> $<INSTALL_INTERFACE:include>)
```

Comment:
There is no need for the `$<INSTALL_INTERFACE>` genex here because the `INCLUDES DESTINATION ...` below does the same thing.